ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -94,6 +94,23 @@ spec:
             fsType: xfs
             devicePaths:
               - /dev/sdb2
+    # manually create PVs if they are not dynamically created by local-storage-operator
+    - fileName: StoragePV.yaml
+      policyName: "config-policy"
+      metadata:
+        name: pv-storage-ptp
+      spec:
+        storageClassName: "example-storage-class-1"
```

Comment:
Is this storage class auto created when this pv is created? If not, then we'd need to add storage class recreation as well in the example. 

Given line 55 and 56 for ptp events is commented (provide example, but optionally configured), we should have line 97-105 commented as well with proper comments.